.github/workflows/preview.yml

```diff
@@ -0,0 +1,56 @@
+name: preview
+
+on:
+  pull_request: ~
+
+permissions:
+  contents: read
+  packages: read
+
+env:
+  NUGET_PACKAGES: ${{ github.workspace }}/.nuget/packages
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Bootstrap Action Workspace
+        id: bootstrap
+        uses: ./.github/actions/bootstrap
+
+      - name: Publish AOT
+        run: ./build.sh publishbinaries
+
+      # we run our artifact directly please use the prebuild
+      # elastic/docs-builder@main GitHub Action for all other repositories!
+      - name: Build documentation
+        run: .artifacts/publish/docs-builder/release/docs-builder --strict
```

Comment:
Should we cache this from the `pr.yml` build and then move the general docs build as part of this workflow ?

In other words we make this depend on the pr workflow.

Reply:
https://github.com/elastic/docs-builder/pull/279/commits/1acb9d9a7795078e7cc77f3b7b04f982ac708a81